session/bootstrap_test.go

```diff
@@ -928,3 +928,118 @@ func TestAnalyzeVersionUpgradeFrom300To500(t *testing.T) {
 	require.Equal(t, 1, row.Len())
 	require.Equal(t, "1", row.GetString(0))
 }
+
+func TestIndexMergeUpgradeFrom300To500(t *testing.T) {
```

Comment:
Add a test for `notBootstrap` to `540`

Reply:
Added in `TestIndexMergeInNewCluster()` .